workspace/Taskfile.yml

```diff
@@ -30,7 +30,7 @@ tasks:
       - './make_backend_tf.sh'
       - touch main.tf
       - task: terraform-command
-        vars: { TF_COMMAND: "init -upgrade", TF_CMD_DIR: "{{.TASKFILE_DIR}}" }
+        vars: { TF_COMMAND: "init --upgrade", TF_CMD_DIR: "{{.TASKFILE_DIR}}" }
```

Comment:
I'm curious about this change. `terraform init --help` lists `-upgrade` as an option.

Reply:
i'm not sure why i did this.  Very strange.  I don't even recall making this modification